database/versions/3b92db7bb2fe_create_initial_tables.py

```diff
@@ -95,6 +95,7 @@ def upgrade():
         sa.Column('file_id', sa.String, nullable=False),
         sa.Column('job_id', sa.String, nullable=False),
         sa.Column('status', validation_event_status_enum, nullable=False),
+        sa.Column('results', JSONB, nullable=True),
```

Comment:
Whoa wait a minute, you're changing the existing migration, that has already been deployed?  That sounds against the spirit of migrations.  Shouldn't this be done in a new migration?   I get that this code hasn't made it past dev yet, perhaps you're thinking it's easier to clear the DB before deploying in this early stage.